test/verify/check-connection

```diff
@@ -986,6 +987,47 @@ server {
         m.start_cockpit(tls=True)
         self.checkCockpitOnProxy()
 
+        # now test with UrlRoot
+        m.write("/etc/cockpit/cockpit.conf", "UrlRoot = cockpit-root\n", append=True)
+        m.execute("systemctl stop cockpit.service")
+        self.sed_file("s_location /_location /cockpit-root_", "/etc/nginx/conf.d/cockpit.conf",
+                      "systemctl restart nginx")
+        self.checkCockpitOnProxy(urlroot="/cockpit-root")
+
+        # get a non-cockpit file from the server
+        m.execute("mkdir -p /srv/www/embed-cockpit")
+        m.upload(["verify/files/embed-cockpit/index.html",
+                  "verify/files/embed-cockpit/embed.js",
+                  "verify/files/embed-cockpit/embed.css"],
+                 "/srv/www/embed-cockpit/")
+        m.execute("if selinuxenabled 2>&1; then chcon -R -t httpd_sys_content_t /srv/www; fi")
+
+        (https_host, https_port) = self.machine.forward["443"].split(':')
+        out = subprocess.check_output(
+            ["curl", "--verbose",
+             "--resolve", f"alice:{https_port}:{https_host}",
+             "--cacert", os.path.join(TEST_DIR, "../src/tls/ca/ca.pem"),
+             f"https://alice:{https_port}/embed-cockpit/embed.css"],
+            stderr=subprocess.STDOUT)
+        self.assertIn(b"HTTP/1.1 200 OK", out)
+        self.assertIn(b"#embed-links", out)
+
+        # embedding
+        b = self.browser
+        b.ignore_ssl_certificate_errors(True)
+        b.open(f"https://{https_host}:{https_port}/embed-cockpit/index.html")
+        b.set_val("#embed-address", f"https://{m.web_address}:{https_port}/cockpit-root")
```

Comment:
This should be `https_host` instead of `m.web_address`. It's the same value, but looks confusing.

Reply:
Fixed.